src/registry/iban/index.tsx

```diff
@@ -12,20 +13,36 @@ export interface FormioIBANProps {
   componentDefinition: IbanComponentSchema;
 }
 
-export const FormioIBAN: React.FC<FormioIBANProps> = ({
-  componentDefinition: {key, label, tooltip, description, validate},
-}) => {
-  return (
-    <TextField
-      name={key}
-      type="text"
-      label={label}
-      tooltip={tooltip}
-      description={description}
-      isRequired={validate?.required}
-      inputMode="text"
-      placeholder=""
+export const FormioIBAN: React.FC<FormioIBANProps> = ({componentDefinition}) => {
+  const {key, label, tooltip, description, validate} = componentDefinition;
+  const sharedProps: Pick<
+    React.ComponentProps<typeof TextField>,
+    'name' | 'label' | 'description' | 'tooltip' | 'isRequired'
```

Comment:
[See comment](https://github.com/open-formulieren/formio-renderer/pull/176/files#r2454504278) of email field